ouroboros-consensus-protocol/src/Ouroboros/Consensus/Protocol/Praos.hs

```diff
@@ -92,17 +91,6 @@ import           Ouroboros.Consensus.Util.Versioned (VersionDecoder (Decode),
 
 data Praos c
 
-class
-  ( Crypto c,
-    DSIGN.Signable (DSIGN c) (OCertSignable c),
-    DSIGN.Signable (DSIGN c) (SL.Hash c EraIndependentTxBody),
```

Comment:
In particular I'm unsure if this was copied right, or the `HASH c` is now wrong? there seems to be a small inconsistency here.

Reply:
I _think_ these are not exactly the same things: `Hash.Hash` takes a hash type which `HASH c`  provides via the type family in `Crypto` but `SL.Hash` might very well take a `c` and do the "magic" of calling `Hash.Hash` with `HASH c`. This is common pattern in ledger base types which BTW introduces an additional dependency to the ledger everywhere those types are used, which is really annoying becuase it increases the dependency surface. 
I failed to see what real value those aliases provide but 🤷